src/utils/functions.ts

```diff
@@ -7,3 +7,27 @@ export function getObjectFromLocalStorage(key: string) {
     return null;
   }
 }
+
+
+export function bubbleSortObject(arr: any[], key: string | number) {
```

Comment:
Do we currently need an implementation of a sorting algorithm or is it to future proof the utils? If it's the second case, we might want to look at other algorithms for sorting for better efficiency.

Reply:
Good question
I'm not sure but I think @mjadach-iv can answer it. In my opinion we should merge this PR if everything is working fine and update the sorting algorithm in the future after the doubts are solved :)

Reply:
In future we will can switch when we will need more performance. Good catch.